### PR TITLE
filestore: add example WASM unknown modules written in Rust and Zig

### PR DESCRIPTION
**Pull request by deadprogram**
This PR adds example modules to the `filestore` example written in Rust using the `wasm32-unknown-unknown` target, and is Zig using the `wasm32-freestanding` target.

### CODE REVIEW

File: filestore/modules/pingrs/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+```
+
+## How to build module
+
+```bash
+cd modules/pingrs
+cargo build --target wasm32-unknown-unknown --release
+cd ../..
+cp ./modules/pingrs/target/wasm32-unknown-unknown/release/pingrs.wasm ./modules/
+```
```
Comment (orsinium) [inline filestore/modules/pingrs/README.md:22]: It would be nice to have a Makefile, or better yet [Taskfile](https://taskfile.dev/) for all this.

**Review by orsinium — APPROVED** [review]
Zig is beautiful ✨

File: filestore/modules/pingrs/README.md (hunk 1 above)
Comment (deadprogram) [reply filestore/modules/pingrs/README.md:22]: I am thinking to add some support for building rust/zig with these settings to the `mecha` command.

**Comment by deadprogram** [comment]
Now merging, thanks for review @orsinium

**Pull request merged**